source/modes/ConsoleCommands.cpp

```diff
@@ -567,6 +568,12 @@ Command::Result cTriggerCompositor(const Command::ArgumentList_t& args, ConsoleI
     return Command::Result::SUCCESS;
 }
 
+Command::Result cStartTileCulling(const Command::ArgumentList_t& args, ConsoleInterface& c, AbstractModeManager&)
+{
+    ODFrameListener::getSingleton().getCameraManager()->mCullingManager->startTileCulling();
```

Comment:
It would be better to use something like ODFrameListener::getSingleton().getCameraManager().startTileCulling();

Using a CullingManager or not should be hidden from the API